src/_includes/v2/footer/socials.html

```diff
@@ -0,0 +1,5 @@
+<p>
+  <a class="font-weight-bold" href="mailto:hello@calitp.org">hello@calitp.org</a>
```

Comment:
I had to add an `aria-label` to get macOS VoiceOver to read this correctly. Without it, it was saying "hello at cal it p dot org" (pronouncing the i and t together as the word "it").


```suggestion
  <a class="font-weight-bold" href="mailto:hello@calitp.org" aria-label="hello at cal i t p dot org">hello@calitp.org</a>
```